src/main/resources/default/extensions/wondergem-page/storage-vfs-modals.html.pasta

```diff
@@ -2,169 +2,180 @@
 
 <i:switch test="@point">
     <i:block name="footer">
-        <w:modal name="select-directory-modal" titleKey="VFSController.selectDirectory">
+        <style>
+            .pagination-controls {
```

Comment:
why inlince css rules that are this generic instead of a scss file included in the biz.scss with properly scoped rules?